### PR TITLE
Proxy and redux toolkit

### PR DESCRIPTION
**Pull request by enver-bisevac**

### CODE REVIEW

File: examples/react-redux/src/ducks/models.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type FeatureFlags = {
+    transmitmmsScheduling?: boolean;
+  };
```
Comment (knagurski) [inline examples/react-redux/src/ducks/models.ts:3]: *SUPER* minor, but indentation is a bit off in this file

File: src/index.ts [hunk 2]
```diff
@@ -293,7 +293,7 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
         })
         .then(() => {
           logDebug('Event stream ready', { storage })
-          eventBus.emit(Event.READY, storage)
+          eventBus.emit(Event.READY, {...storage})
```
Comment (knagurski) [inline src/index.ts:296]: Nice and simple 👍🏻

**Review by knagurski — COMMENTED** [review]
Looks the part, just needs the `package.json` and `package-lock.json` version bump

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harnessio/ff-javascript-client-sdk",
-  "version": "1.4.9",
+  "version": "1.4.10",
```
Comment (knagurski) [inline package.json:3]: Looks like the `package-lock.json` file is still sitting at `1.4.8`

**Review by knagurski — COMMENTED** [review]
Sorry, I thought I hit go on that comment yesterday!

File: package-lock.json [hunk 4]
```diff
@@ -1,8 +1,519 @@
 {
   "name": "@harnessio/ff-javascript-client-sdk",
-  "version": "1.4.8",
-  "lockfileVersion": 1,
+  "version": "1.4.10",
+  "lockfileVersion": 2,
```
[the remark this answers is not in the document]
Comment (enver-bisevac) [reply package-lock.json:4]: No I just run npm

**Pull request merged**